src/commit.ts

```diff
@@ -108,3 +111,57 @@ export function groupCommits(commits: CommitHeader[]): CommitGroup[] {
 export const formatCommitTime = (t: number): string => {
   return new Date(t * 1000).toUTCString();
 };
+
+export function groupCommitsByWeek(commits: CommitHeader[]): CommitGroup[] {
+  const groupedCommits: CommitGroup[] = [];
+  let groupDate: Date | undefined = undefined;
+
+  commits = commits.sort((a, b) => {
+    if (a.committerTime > b.committerTime) {
+      return -1;
+    } else if (a.committerTime < b.committerTime) {
+      return 1;
+    }
+
+    return 0;
+  });
```

Comment:
Note that this is sorting in reverse order, ie. from latest commit to earliest. Is this intended?

Reply:
I tried different ways of building the graph and the easiest way I found was to start the graph from the right side = latest, filling up the weeks with no commits until the latest and from there go back to the earliest.